doc/release/release_0.19.rst

```diff
@@ -1,3 +1,37 @@
+scikit-image 0.19.1
+===================
+
+We're happy to announce the release of scikit-image v0.19.1!
+
+This is a small bug fix release that resolves a couple of backwards compatibility issues and a couple of issues with the wheels on PyPI. Specifically, Apple M1 wheels on PyPI were broken in 0.19.0, but should now be repaired. Wheel sizes are also greatly reduced relative to 0.19.0 by stripping debug symbols from the binaries and making sure that Cython-generated source files are not bundled in the wheels.
```

Comment:
```suggestion
This is a small bug fix release that resolves a couple of backwards compatibility issues and a couple of issues with the wheels on PyPI. Specifically, OSX M1 wheels on PyPI were broken in 0.19.0, but should now be repaired. Wheel sizes are also greatly reduced relative to 0.19.0 by stripping debug symbols from the binaries and making sure that Cython-generated source files are not bundled in the wheels.
```
(more standard)

Reply:
Good catch, maybe make it consistent with the wheel arch too so: Apple M1 (arm64)

Reply:
Actually, I think it's important to mention the M1 wheel is for macOS 12 (Monterey):
`export MACOSX_DEPLOYMENT_TARGET=12.0`
It's been out for nearly 2 mo, but probably not everyone runs it.

Reply:
Thanks, the language has been updated to mention this